### PR TITLE
Indicators for staff and contractor expenses

### PR DESCRIPTION
**Pull request by longhotsummer**
Also:

* draw charts based on data attributes to simplify things
* the data is injected automatically via the profileData var in javascript.

### CODE REVIEW

**Comment by longhotsummer** [comment]
@jbothma ok, ready to review and/or merge

**Comment by longhotsummer** [comment]
Bleh, big change, sorry about that.

File: scorecard/profile_data.py [hunk 1]
```diff
@@ -22,58 +21,361 @@
...
             }
         elif query['query_type'] == 'model':
             url = self.API_URL + query['cube'] + '/model'
             params = {}
-        return self.session.get(url, params=params, verify=False)
 
-    def response_to_results(self, api_response, query):
-        api_response.result().raise_for_status()
-        response_dict = api_response.result().json()
+        response = self.session.get(url, params=params, verify=False)
```
Comment (jbothma) [inline scorecard/profile_data.py]: I suspect calling get() and result() in the same loop will lose the benefit of futures. That's why we do one loop of get()s and another loop of result()s

Comment (longhotsummer) [inline scorecard/profile_data.py]: Oh, that makes sense. I didn't understand why it was done like that.

**Comment by jbothma** [comment]
Nice. I don't think I caught all the changes but I like what's happening here

**Comment by longhotsummer** [comment]
Yeah, a big pull request like this isn't helpful. My bad.

**Pull request merged**